src/context/config.test.tsx

```diff
@@ -37,19 +33,16 @@ describe("useConfigContext", () => {
   });
 
   it("should load saved config from async storage properly", async () => {
-    expect.assertions(5);
+    expect.assertions(2);
     const { queryByTestId } = render(
       <ConfigContextProvider>
         <TestComponent />
       </ConfigContextProvider>
     );
-    expect(queryByTestId("loading-view")).not.toBeNull();
-    expect(queryByTestId("printed-network")).toBeNull();
-
-    await waitForElementToBeRemoved(() => queryByTestId("loading-view"));
-    expect(queryByTestId("loading-view")).toBeNull();
-    expect(queryByTestId("printed-network")).not.toBeNull();
-    expect(queryByTestId("printed-network")).toHaveTextContent("mainnet");
+    await wait(() => {
+      expect(queryByTestId("printed-network")).not.toBeNull();
+      expect(queryByTestId("printed-network")).toHaveTextContent("mainnet");
+    });
```

Comment:
Hmm, the existing test tested the idea that the loading view will be shown briefly, what are some ways we can ensure this functionality?

Reply:
Hmm, I realized that we were using useState hooks to show screens, however, useState(boolean) technically only holds 2 states (T/F), undefined is not a valid value. If we want to show a loading view, I would suggest we have an isLoading state as well!

Reply:
Implemented an isLoaded state to show loading screen state 👍